### DISCUSSION TITLE
Features where properties are null are not shown

### DISCUSSION THREAD

**Discussion by Sverre-Iversen** [ldproxy/ldproxy#896] [Configuration] [answered]
Hi, is it possible to force display of properties with null values?
Kind Regards
Sverre Iversen

**Comment by cportele** [comment, marked as answer]
Currently not, these are always omitted.

**Reply by Sverre-Iversen** [reply]
OK, I can alternatively add a null substitute

**Comment by zvfcs** [comment]
I also encountered this problem - or rather, a user encountered it when trying to use FME to fetch data from the API. When you add a feature type as a reader in FME, it uses the first item to determine what properties it contains. So when one or more are omitted because of NULL values, the properties don't exist according to FME.

@Sverre-Iversen: For your solution, where did you implement it? I can't seem to find a "default if null" type of configuration option.

**Reply by cportele** [reply]
With v4.0 a new transformation [`removeNullValues`](https://docs.ldproxy.net/providers/details/transformations.html) was introduced to suppress the removal of `null` values. The following should work:

```yaml
- buildingBlock: GEO_JSON
  transformations:
    '*':
    - removeNullValues: false
```

If FME also supports other formats than GeoJSON, another option would be to access the data in a format that requires explicit `null` values, e.g., FlatGeobuf.

**Reply by zvfcs** [reply]
Ah - that's nice. I'm on version 3.3.4 because of performance issues with later versions and a sorting issue (that as I understand it will be fixed soon). Hopefully I'll get around the performance issue soon, so I can upgrade. My hope is that a newer Postgresql version will help.

Not sure about how to request for other formats with FME, but I'll look into it.
